### DISCUSSION TITLE
Support for 2-bit bitmap

### DISCUSSION THREAD

**Discussion by gingerbeardman** [ImageMagick/ImageMagick#7057] [Contribute]
### Is your feature request related to a problem? Please describe.

- old painting app JINZO Paint (PocketPC, Windows CE) saves images as 2-bit, with file extension .2bp

`magick` complains:
```
% magick shartoise.2bp -colors 4 -type palette shartoise.png
magick: unsupported bits per pixel `shartoise.2bp' @ error/bmp.c/ReadBMPImage/987.
```

### Describe the solution you'd like

- support for reading and writing such files

### Describe alternatives you've considered

GOOD
- GraphicConverter 11 opens them (as-is)
- macOS (Preview, QuickLook, Acorn, etc) opens them (with extension .2bp.bmp or .bmp)

BAD
- Photoshop 2023 does not open them

### Additional context

Example files: [Archive.zip](https://github.com/ImageMagick/ImageMagick/files/14003214/Archive.zip)
- shartoise.2bp (problem)
- shartoise.bmp (works fine)

`file` reports:
```
shartoise.2bp: PC bitmap, Windows 3.x format, 480 x 296 x 2, image size 35520, cbSize 35590, bits offset 70
shartoise.bmp: PC bitmap, Windows 3.x format, 480 x 296 x 4, image size 71040, cbSize 71158, bits offset 118
```

reference image (PNG)
![shartoise](https://github.com/ImageMagick/ImageMagick/assets/49612/631671ca-87fc-46d6-9041-e1148a31ca2a)

**Comment by gingerbeardman** [comment]
@urban-warrior curious to know why it's moved to a discussion?

**Reply by dlemstra** [reply]
All our up-for-grabs issues are moved to the discussions since they are not really an "issue" but more a feature request.

**Reply by gingerbeardman** [reply]
Thanks for the info! I'll take a shortcut next time now that I know this.
